### DISCUSSION TITLE
Will my tauri desktop app still work if a user doesn't have any browsers installed on their machine?

### DISCUSSION THREAD

**Discussion by nooblyf** [tauri-apps/tauri#10009] [Q&A] [answered]
Haven't been able to test it because i'm on mac and it is not possible to delete Safari afaik

**Comment by FabianLars** [comment, marked as answer]
Yes, the webviews are seperate from browsers on all platforms. On some platforms under some conditions webviews and browsers will share some of their components (== saving disk space) but uninstalling browsers will not remove the webviews that tauri uses.

**Reply by nooblyf** [reply]
Got it, thanks!
